sdk/keyvault/azure-security-keyvault-keys/test/ut/recordings/Crypto-KeyVaultKeyClientWithParam.GetCryptoFromKeyRemoteEncrypt-0.json

```diff
@@ -0,0 +1,85 @@
+{
+  "networkCallRecords": [
+    {
+      "Headers": {
+        "content-type": "application/json",
+        "user-agent": "azsdk-cpp-keyvault-keys/4.3.0-beta.1 (Linux 5.4.0-1063-azure x86_64 #66~18.04.1-Ubuntu SMP Thu Oct 21 09:59:28 UTC 2021)",
+        "x-ms-client-request-id": "4020f41e-00f1-40db-5648-fd38c8b90c02"
+      },
+      "Method": "POST",
+      "Response": {
+        "BODY": "{\"key\":{\"kid\":\"https://cpp-recordings.vault.azure.net/keys/GetCryptoFromKeyRemoteEncrypt-0/9faa062049c24d29b9c0506fb09d80df\",\"kty\":\"RSA\",\"key_ops\":[\"encrypt\",\"decrypt\",\"sign\",\"verify\",\"wrapKey\",\"unwrapKey\"],\"n\":\"wjuDjBiMiSslq6_e_SFhbfZfz4R6PKrgDobCDiQ-Bsn_jSrnM1QNlImzIn5YL04iNJylrtfPeyUWN7MutXAHSrjEufudKIteF60yn131miaee4OP1VEXGBJYUG2IMbS5vhOC43MxHm3mDG_7Y22cel0iWBYqiqorMCR5l06aTAKEl3k3MO0oZQ4qottX1_K8LyAvYm4byjNQiPOaM45wPWh4YRfJZpv15o-U4VSMHJ5tDqj1RVzS4MALXlwPKTq3-B02ec-Cz4aMmn77J68x1WVV4BTtKmm5DW8-OYVpQw4Rrvlf9CWpj2LkJ_bqAKIm7jAd8AItwcn9asBa4tYh5Q\",\"e\":\"AQAB\"},\"attributes\":{\"enabled\":true,\"created\":1638402259,\"updated\":1638402259,\"recoveryLevel\":\"Recoverable+Purgeable\",\"recoverableDays\":90}}",
```

Comment:
Does this recording name follow the pattern other languages use?

Reply:
does not really matter.  The name is just the name of the account I used to created the recordings and then I deleted that account.